packages/eslint-plugin/src/lib/util.ts

```diff
@@ -0,0 +1,144 @@
+import { Rule } from 'eslint';
+import { ParserServices } from '@typescript-eslint/typescript-estree';
```

Comment:
shouldn't this come from `eslint`? [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a3beba76502aa411eabb050ef1573e8ffbfff10c/types/eslint/index.d.ts#L215) (god awful type there but maybe it'll get better one day)

Reply:
Nope! The `parserServices` property is free for the parser to implement however they like, so they will never type it any strong than `any`.

I specifically exported that interface from `typescript-estree` so that I could provide a strict type for the plugin.